### PR TITLE
Remove pread_exact from the driver struct

### PR DESCRIPTION
**Pull request by rminnich**
It makes more sense as as function.

Signed-off-by: Ronald G. Minnich <rminnich@gmail.com>

### CODE REVIEW

File: src/drivers/model/src/lib.rs [hunk 1]
```diff
@@ -17,23 +17,37 @@ pub trait Driver {
     fn pread(&self, data: &mut [u8], pos: usize) -> Result<usize>;
     /// Positional write. Returns number of bytes written.
     fn pwrite(&mut self, data: &[u8], pos: usize) -> Result<usize>;
+    /// Control the device.
+    /// The data is an arbitrary string.
+    /// This function should not be called directly; rather, use the ctl macro,
+    /// which ensures that the parameter is valid. The ctl function is allowed
+    /// to ignore the request. It will return an error only in the event
+    /// of a real error. The result should included any disambuguiting information
```
Comment (fotonick) [inline src/drivers/model/src/lib.rs:25]: s/included/include/
s/disambuguiting/disambiguating/

File: azure-pipelines.yml [hunk 2]
```diff
@@ -70,10 +70,10 @@ jobs:
   - script: |
       (
         cd src/mainboard/emulation/qemu-fsp
-        timeout 120s make run | tee serial
-        grep 'Running payload' serial
+        echo 'DO NOT timeout 120s make run | tee serial'
```
Comment (fotonick) [inline azure-pipelines.yml:73]: I presume you'll put these back, as they're unrelated to the nominal change.

**Review by fotonick — COMMENTED** [review]
I don't really understand why you'd remove it from the Trait. For example, `std::io::Read` works such that once you write the basic `read()` function, other things just work, including `read_exact()`. `pread_exact()` feels idiomatic as a Trait method.

File: azure-pipelines.yml (hunk 2 above)
Comment (rminnich) [reply azure-pipelines.yml:73]: I'm pretty sick of the FSP build failing, I don't want to put this back until it works.

File: src/drivers/model/src/lib.rs (hunk 1 above)
Comment (rminnich) [reply src/drivers/model/src/lib.rs:25]: It doesn't feel right to me. But I think you're right. I am going to close this.

**Pull request closed without merging**